src/queries/models/rewards/getRewardsActivitiesModelCreator.ts

```diff
@@ -1,4 +1,15 @@
+import { ActivityConfig } from 'modules/rewards/Rewards.constants';
 import { RewardsAcitivitesResponse } from 'queries/types';
 
-export const getRewardsActivitiesModelCreator = (response: RewardsAcitivitesResponse): RewardsAcitivitesResponse =>
-  response;
+export const getRewardsActivitiesModelCreator = (response: RewardsAcitivitesResponse): RewardsAcitivitesResponse => {
+  response.activities = response.activities.map((activity) => {
+    const config = ActivityConfig[activity.id];
```

Comment:
@abhishek-01k needs to be done using activityType and not id, since id could be different for backend of different envs